Sorting/quick-sort/quickSort.java

```diff
@@ -0,0 +1,46 @@
+import java.io.*;
+
+public class Quick_Sort {
+  public static void main(String[] args) {
+    int[] arr = {5, 7, 3, 10, 2000, 9, 19, 16};
+
+    // sort the array
+    Quick_Sort(arr, 0, arr.length - 1);
+
+    // display the array
+    for (int i = 0; i < arr.length; i++)
+      System.out.print(arr[i] + ", ");
+    System.out.println("End");
```

Comment:
How about using spaces instead of commas here? Then it won't look odd if you don't use "End" at last.

So for example, outputting "`1 4 10 12`" instead of "`1, 4, 10, 12, End`"?

Reply:
Sure, updating this